ecl/dedicated_hypervisor/v1/vcenter.py

```diff
@@ -68,3 +71,19 @@ def update(self, session, vcenter_id, password, license_id):
         )
         self._translate_response(resp, has_body=True)
         return self
+
+
+class LinkLocalAddresses(resource2.Resource):
+    base_path = '/vcenters/addresses'
+    resources_key = 'addresses'
+    resource_key = 'address'
+    service = dedicated_hypervisor_service.DedicatedHypervisorService()
+
+    # capabilities
+    allow_list = True
+
+    # Properties
+    #: IP address.
+    address = resource2.Body('address')
```

Comment:
```suggestion
    address = resource2.Body('address', alternate_id=True)
```
パラメーターにIDがないときは、代替としてどのパラメーターを IDとみなすか指定が必要です。
IPアドレスならば一意のため、addressに `alternate_id=True` を指定してください。